eosio/utils/create_accounts.sh

```diff
@@ -66,7 +66,7 @@ if [ ! "$bridge_account" ]; then
         bridge_account="$(name_maker)"
         store_env bridge_account $bridge_account
 
-        bbs_active_public_key="$(cat state.env | grep "${bbs_account}_active_public_key" | sed 's/.*=//' | sed 's/^.//;s/.$//')"
+        bbs_active_public_key="$(cat ../state.env | grep "${bbs_account}_active_public_key" | sed 's/.*=//' | sed 's/^.//;s/.$//')"
```

Comment:
I do not like this. Can't `store_env` take care of this and set the variable while writing it to the file?

Reply:
only the public key of bbs is read from state.env, so I don’t see why to change store_env.